<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZPR_STEGANOClass</class>
 <widget class="QMainWindow" name="ZPR_STEGANOClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZPR_STEGANO</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="encryptButton">
      <property name="text">
       <string>encrypt</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="decryptButton">
      <property name="text">
       <string>decrypt</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openFileButton">
      <property name="text">
       <string>openFile</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="openFileLabel">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>choosenFile</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveFileButton">
      <property name="text">
       <string>save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="saveFilepathLabel">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>filepath to save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="methodWidgetLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
